cli/src/command/config_transaction_create.rs

```diff
@@ -0,0 +1,304 @@
+                        multisig,
+                        rent_payer: transaction_creator,
+                        transaction: transaction_pda.0,
+                        system_program: solana_sdk::system_program::id(),
+                    }
+                    .to_account_metas(Some(false)),
+                    data: ConfigTransactionCreateData {
+                        args: ConfigTransactionCreateArgs {
+                            actions: vec![config_action.unwrap()],
+                            memo,
+                        },
+                    }
+                    .data(),
+                    program_id,
+                },
+                Instruction {
+                    accounts: ProposalCreateAccounts {
+                        creator: transaction_creator,
+                        multisig,
+                        rent_payer: transaction_creator,
+                        proposal: proposal_pda.0,
+                        system_program: solana_sdk::system_program::id(),
+                    }
+                    .to_account_metas(Some(false)),
+                    data: ProposalCreateData {
+                        args: ProposalCreateArgs {
+                            draft: false,
+                            transaction_index,
+                        },
+                    }
+                    .data(),
+                    program_id,
+                },
+            ],
+            &[],
+            blockhash,
+        )
+        .unwrap();
+
+        let transaction = VersionedTransaction::try_new(
+            VersionedMessage::V0(message),
+            &[&*transaction_creator_keypair],
+        )
+        .expect("Failed to create transaction");
+
+        match rpc_client.send_and_confirm_transaction(&transaction).await {
+            Ok(signature) => {
+                progress.finish_with_message(format!("Transaction confirmed: {}\n\n", signature));
+            }
+            Err(err) => {
+                progress.finish_and_clear();
+
+                if let ClientErrorKind::RpcError(RpcError::RpcResponseError {
+                    data:
+                        RpcResponseErrorData::SendTransactionPreflightFailure(
+                            RpcSimulateTransactionResult {
+                                logs: Some(logs), ..
+                            },
+                        ),
+                    ..
+                }) = &err.kind
+                {
+                    println!("Simulation logs:\n\n{}\n", logs.join("\n").yellow());
+                };
+
+                return Err(eyre!(format!(
+                    "Transaction failed: {}",
+                    err.to_string().red()
+                )));
+            }
+        }
+
+        println!(
+            "✅ Created Config Transaction. Signature: {}",
+            transaction.signatures[0]
+        );
+        Ok(())
+    }
+}
+
+fn parse_action(action_str: &str) -> Result<ConfigAction, String> {
```

Comment:
maybe worth making 'action' an enum that implements clap's ValueEnum for better help description.